### ISSUE TITLE
fastboot?

### ISSUE THREAD

**Issue by midget2000x** [DockYard/ember-skeleton#12]
I'm thinking this is not compatible with fastboot, correct?  I'm getting:

` ReferenceError: Image is not defined`

in reference to:

`var img = new Image();`

I kind of understand why it wouldn't be fastboot compatible.  Just checking tho!  And thanks!

**Comment by bcardarella** [comment]
Yes, I agree. However, I am not sure what the fastboot render behavior should be. Do we want fastboot to always render the original image or the placeholder? I suspect the placeholder, then when Ember steps in we get the normal behavior. Thoughts?

**Comment by midget2000x** [comment]
Seems reasonable to me.  However...I am still wrapping my head around how fastboot works (and ember for that matter)!
